.github/workflows/build-test.yaml

```diff
@@ -50,7 +50,10 @@ jobs:
       - uses: "actions/setup-go@v3"
         with:
           go-version: "^1.17"
-      - uses: "authzed/actions/docker-build@main"
+      - uses: "docker/build-push-action@v1"
```

Comment:
probably need to rename this job

`image-build` => `docker-tests`
`Build Container Image` => `Build & Test Container Image`

Reply:
If we do, it'll break the CI submit checks, so I left it as is (unless we should change those too)

Reply:
It's really easy to change them. I'd rather spend two seconds in the GitHub settings than live with names and descriptions of things that don't make sense.